### ISSUE TITLE
Entity Recognition example 3 fails

### ISSUE THREAD

**Issue by vaulttech** [explosion/spaCy#803]
I am checking out the documentation on Entity Recognition. It crashes when I try to run this line:

https://github.com/vaulttech/spaCy/blob/master/website/docs/usage/entity-recognition.jade#L57

I was initially getting a simple

    NameError: name 'Span' is not defined

which I concluded was going to be obvious to solve by simply referring to `spacy.Span` instead of `Span`. This also didn't work; but I noticed `doc.ents[0]` would have an element of type Span, so I ran `type(doc.ents[0])` and got `spacy.tokens.span.Span` as result. I then tried changing the line into:

    docs.ents = [spacy.tokens.span.Span(0, 1, label='GPE')]

But I still get an error: (the stuff below are the output of ipython)

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-41-f39cf651a81b> in <module>()
----> 1 docs.ents = [spacy.tokens.span.Span(0, 1, label='GPE')]

c:\python27\lib\site-packages\spacy\tokens\span.pyx in spacy.tokens.span.Span.__cinit__ (spacy/tokens/span.cpp:3246)()
     19 cdef class Span:
     20     """A slice from a Doc object."""
---> 21     def __cinit__(self, Doc doc, int start, int end, int label=0, vector=None,
     22                   vector_norm=None):
     23         '''Create a Span object from the slice doc[start : end]

TypeError: __cinit__() takes at least 3 positional arguments (2 given)
```

I also tried changing the line into:

    docs.ents = [spacy.tokens.span.Span(docs, 0, 1, label='GPE')]

but it seems that the `label` argument is supposed to be an integer, instead of a string:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-46-2a8dab63bbd4> in <module>()
----> 1 docs.ents = [spacy.tokens.span.Span(docs, 0, 1, label='GPE')]

c:\python27\lib\site-packages\spacy\tokens\span.pyx in spacy.tokens.span.Span.__cinit__ (spacy/tokens/span.cpp:3283)()
     19 cdef class Span:
     20     """A slice from a Doc object."""
---> 21     def __cinit__(self, Doc doc, int start, int end, int label=0, vector=None,
     22                   vector_norm=None):
     23         '''Create a Span object from the slice doc[start : end]

TypeError: an integer is required
```
Any idea?

I am using spaCy in a Windows 7 Ultimate, Service Pack 1, 64-bit, with Python 2.7.13. I installed spaCy today following the tutorial (i.e., with `pip install -U spacy`). It's version in my computer is 1.6.0.

**Comment by honnibal** [comment]
Thanks! Fixed.

**Comment by honnibal** [comment]
The answer is that the `Span` is like a slice object, so you need the `Doc` object that the `Span` is a slice of as an argument. The string/int thing is unfortunate. I'll get that fixed in 2.0.
